<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurateWorkspace</class>
 <widget class="QDialog" name="ConfigurateWorkspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurate Workspace</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(171, 171, 171);
background-color: rgb(29, 29, 29);</string>
  </property>
  <widget class="QDialogButtonBox" name="dialog_buttons">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>236</y>
     <width>144</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_service">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>90</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Service:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_service">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>91</y>
     <width>237</width>
     <height>17</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_database">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>66</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_pass">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>147</y>
     <width>237</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="examin_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>44</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string>Examin</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_db_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>175</y>
     <width>237</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_database">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>67</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_csv">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>22</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>CSV</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_database_name">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>174</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Database name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_table_name">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>202</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Table name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>117</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_csv">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>23</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_csv">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>239</width>
     <height>18</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_table_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>203</y>
     <width>237</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_user">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>119</y>
     <width>237</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>147</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurateWorkspace</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurateWorkspace</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
